src/holofuel/components/FlashMessage/FlashMessage.test.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import { render, fireEvent, act } from '@testing-library/react'
+import wait from 'waait'
+import FlashMessage from './FlashMessage.js'
+import useFlashMessageContext, { FlashMessageProvider } from 'holofuel/contexts/useFlashMessageContext'
+
+// Think about ways to refactor this to test fewer implementation details.
```

Comment:
These tests are *fine*, but they test implementation details. Because this is a global component, avoiding that is tricky and requires basically mounting the whole app. I went for simplicity and readability of tests over purity of testing philosophy in this case.